Source/ZoomNet/Models/Registrant.cs

```diff
@@ -131,15 +128,19 @@ public class Registrant
 		/// <summary>
 		/// Gets or sets the date and time when the registrant was created.
 		/// </summary>
-		/// <value>The registrant created time.</value>
 		[JsonPropertyName("create_time")]
 		public DateTime CreatedOn { get; set; }
 
 		/// <summary>
 		/// Gets or sets the URL for this registrant to join the meeting.
 		/// </summary>
-		/// <value>The join URL.</value>
 		[JsonPropertyName("join_url")]
 		public string JoinUrl { get; set; }
+
+		/// <summary>
+		/// Gets or sets the participant pin code used to authenticate audio participants before they join the meeting.
+		/// </summary>
+		[JsonPropertyName("participant_pin_code")]
+		public long ParticipantPinCode { get; set; }
```

Comment:
You said that this pin code "is available in some cases" implying that sometimes it's not available. Therefore shouldn't this property be nullable? Meaning something like `long? ParticipantPinCode`.

What do you think?